packages/navitem/src/react/common.tsx

```diff
@@ -12,6 +12,8 @@ const styles = {
   bar: (props: {
     alignment: ValueOf<typeof alignments>
     selected: boolean
+    // eslint-disable-next-line camelcase
+    UNSAFE_stylesFor: unknown
   }) => {
     const isHorz = (props.alignment = alignments.horizontal)
     const isVert = (props.alignment = alignments.vertical)
```

Comment:
These look fishy to me.  Seems like this should be a test (`==`) instead of assignment (`=`).

Reply:
ha. good catch. 🤦